tools/topology/topology2/include/pipelines/cavs/mixout-gain-dai-copier-playback.conf

```diff
@@ -47,7 +47,7 @@ Class.Pipeline."mixout-gain-dai-copier-playback" {
 	Object.Widget {
 		mixout."1" {}
 		copier."1" {
-			type dai_in
+			type dai_out
```

Comment:
I am not sure about this one @plbossart . I think you made too many changes in this PR that are not correct and you've got failures left, right & center. It might be a good idea to go step by step and separate the clean ups one by one